src/Data/Array.purs

```diff
@@ -556,21 +827,26 @@ span p arr =
 -- | Group equal, consecutive elements of an array into arrays.
 -- |
 -- | ```purescript
--- | group [1,1,2,2,1] == [[1,1],[2,2],[1]]
+-- | group [1,1,2,2,1] == [NonEmpty 1 [1], NonEmpty 2 [2], NonEmpty 1 []]
 -- | ```
 group :: forall a. Eq a => Array a -> Array (NonEmpty Array a)
 group xs = groupBy eq xs
 
 -- | Sort and then group the elements of an array into arrays.
 -- |
 -- | ```purescript
--- | group' [1,1,2,2,1] == [[1,1,1],[2,2]]
+-- | group' [1,1,2,2,1] == [NonEmpty 1 [1,1],NonEmpty 2 [2]]
 -- | ```
 group' :: forall a. Ord a => Array a -> Array (NonEmpty Array a)
 group' = group <<< sort
 
 -- | Group equal, consecutive elements of an array into arrays, using the
 -- | specified equivalence relation to detemine equality.
+-- |
+-- | ```purescript
+-- | groupBy (\a b -> odd a && odd b) [1, 3, 2, 4, 3, 3] = [NonEmpty 1 [3], NonEmpty 2 [] , NonEmpty 4 [], NonEmpty 3 [3]]
```

Comment:
Can you please break this up over multiple lines? I don't think this will look great rendered as HTML. (You can check with `pulp docs -- --format html`)